contracts/dao/proposal/cwd-proposal-multiple/src/query.rs

```diff
@@ -41,3 +42,9 @@ pub struct VoteListResponse {
 pub struct ConfigResponse {
     pub config: Config,
 }
+
+/// A list of proposals returned by `ProposalFailedExecutionError`.
+#[cw_serde]
+pub struct FailedProposalErrors {
+    pub errors: Vec<FailedExecutionError>,
+}
```

Comment:
yeah, but they are different.
I'm not sure about it, since we have a lot of copypaste like this for single/multiple/subdao code and it's okay there.

Reply:
what exactly would be inconsistent? I can see that, speaking of query response models, at least single choice proposal and multi choice proposal contracts have slightly different models for similar queries (e.g. `ListProposals`, `Proposal`, `GetVote`), whereas having same models (`ProposalCreationPolicy`, `ProposalHooks`) exported from shared packages